stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/statement/CallStatement.java

```diff
@@ -120,10 +120,15 @@ public void execute() {
                 cleanupResources(resultSets, stmt, conn, !isInTransaction);
                 context.setReturnValues();
             }
+        } catch (SQLException e) {
```

Comment:
We cannot simply stop by checking SQLException at the outer layer. There are multiple places in [1] where SQLExceptions are re-thrown as BallerinaException. So we need to change them as well.

[1] https://github.com/ballerina-platform/ballerina-lang/blob/master/stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/statement/AbstractSQLStatement.java